tests/Doctrine/Tests/DBAL/Types/JsonArrayTest.php

```diff
@@ -43,12 +43,12 @@ public function testReturnsSQLDeclaration()
 
     public function testJsonNullConvertsToPHPValue()
     {
-        self::assertSame(array(), $this->type->convertToPHPValue(null, $this->platform));
+        self::assertEmpty($this->type->convertToPHPValue(null, $this->platform));
```

Comment:
`self::assertSame()` must be kept here since `Type#convertToPHPValue()` has a `mixed` return and we must ensure that it's an array with no entry (not only something empty).